word_counter.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
+
+class WordCounter
+  def load_file(filename)
+    @words = {}
+    @text = File.read(filename)
+    @text.split(/\b/).grep(/[a-zA-Z]/).map { |word| 
+      @words[word] = @words.fetch(word, 0) + 1 
+    }
+    
+    #print output
+    @words.sort_by(&:last).reverse.first(10).each { |key, value|
+      puts "#{value} - #{key}"
+    }
+
+  end
```

Comment:
I recommend using `do...end` for blocks that aren't on one line (with the method call).